server/controllers/userController.js

```diff
@@ -12,20 +12,20 @@ module.exports = {
     });
   },
 
-  createUser: function (req, res) {
-    var userData = {
-      firstName: req.body.firstName,
-      lastName: req.body.lastName,
-      email: req.body.email,
-    };
-    User.create(userData)
-    .then(function (user) {
-      res.status(201).json(user);
-    })
-    .catch(function (err) {
-      console.error (err);
-    });
-  },
+  // createUser: function (req, res) {
```

Comment:
Doesn't this info have to be populated by the info stored in the session? It was my understanding that the client-side doesn't have access to it, so it wouldn't be carried in on req.body.